.applier/inventory/group_vars/all.yml

```diff
@@ -4,18 +4,28 @@ build_vars:
   GIT_REF: master
   OUTPUT_IMAGE_NAME: omp-backend
   OUTPUT_IMAGE_TAG: latest
-  BUILDER_IMAGE_NAME: quay.io/quarkus/ubi-quarkus-native-s2i
-  BUILDER_IMAGE_TAG: 19.2.1
+  BUILDER_IMAGE_NAME: registry.redhat.io/openjdk/openjdk-8-rhel8
+  BUILDER_IMAGE_TAG: 1.1
+  BUILDER_IMAGE_PULL_SECRET: builder-image-pull-secret
 
 deploy_vars:
   NAME: omp-backend
   IMAGE_NAME: omp-backend
   IMAGE_TAG: latest
   JWT_PUBLIC_KEY_URL: http://sso-cluster-internal:8080/auth/realms/omp/protocol/openid-connect/certs
+  GIT_REPO_URL: git@gitlab.com:my-org/omp-repo.git
+  GIT_PRIVATE_KEY_PATH: /identity/ssh-privatekey
+  GIT_SECRET_NAME: git-key
 
 openshift_cluster_content:
 - object: omp-backend
   content:
+# Uncomment this if you would like to define your image pull secret here, and then DO NOT push this back to a public repository.
```

Comment:
I wonder i there is a way to not have to uncomment - Like a default exclude_tags (i don't know if this exists).

Do we have a specific need to support these 3 variations?